contracts/RoleAccessControl.sol

```diff
@@ -111,6 +114,8 @@ abstract contract RoleAccessControl is RoleMembership {
 
     function revokeSuperUserRole(address account) external onlySuperUserRole {
         _revokeGlobalRole(Roles.SUPER_USER, account);
+        require(_superUserCounter > 1, "revoking last SuperUser");
```

Comment:
issue: align the error message with the parent contract error messages https://github.com/windranger-io/windranger-treasury/blob/main/contracts/RoleMembership.sol

As there just a single case, prefixing the contract name, or an abbreviation would be fine.

Reply:
Okey @CjHare, you mean an error msg "RoleAccess: last SuperUser", something like that, that is okey with the linter?

Reply:
Yeah something like `RoleAccessControl: no revoking last SuperUser`

Shorter version can include abbreviations for the contract
`RAC: no revoking last SuperUser`

Reply:
okey abbreviations gotcha.

Reply:
error msg improved @CjHare